client/helpers/files.ts

```diff
@@ -21,11 +21,10 @@ export async function uploadFileIfNotExists(
   const existingFile = suppliedId
     ? await getBySuppliedId<FileMetadata, FileList>(
         () =>
-          args.client.files.getFiles(
-            undefined,
-            1,
-            encodeIfNotEncoded(suppliedId)
-          ),
+          args.client.files.getFiles({
+            pageSize: 1,
+            filterSuppliedId: encodeIfNotEncoded(suppliedId),
+          }),
```

Comment:
Prevents users from having to pass `undefined` for optional params they don't want to specify.